security/cmd/istio_ca/main.go

```diff
@@ -160,11 +158,10 @@ func init() {
 	flags.StringVar(&opts.listenedNamespaces, "listened-namespaces", metav1.NamespaceAll,
 		"Select the namespaces for the Citadel to listen to, separated by comma. If unspecified, Citadel tries to use the ${"+
 			cmd.ListenedNamespaceKey+"} environment variable. If neither is set, Citadel listens to all namespaces.")
+	flags.BoolVar(&opts.manageUnlabeled, "manage-unlabeled", true, "Determines whether Citadel"+
```

Comment:
I think we discussed not adding flags, but instead using a config file that can be updated without restart. 
Adding values.yaml or the citadel subsection to a config map would be better than flag, and avoid the complexity of reading config, installer adding flag, restart, etc.

I believe Galley already does this with viper package.

Reply:
Agreed this would be ideal, but should it perhaps be done in a separate PR? If we decide to make Citadel support hot reloading config, we would want to move other flags into the config map as well. Might be a task worthy of its own review